projects/plugins/jetpack/extensions/plugins/publicize/hooks/use-publicize-config/index.js

```diff
@@ -28,7 +26,6 @@ export default function usePublicizeConfig() {
 		isPublicizeEnabled,
 		togglePublicizeFeature,
 		isRePublicizeFeatureAvailable: available,
-		isRePublicizeFeatureUpgradable:
-			isUpgradable( republicizeFeatureName ) && isUpgradeNudgeEnabled(),
+		isRePublicizeUpgradableViaUpsell: isUpgradable( republicizeFeatureName ),
```

Comment:
> From what I can tell, this is the approach @retrofox took: isRePublicizeUpgradableViaUpsell relies on isUpgradeNudgeEnabled():

I discovered that we don't need to use `isUpgradeNudgeEnabled`. The `isUpgradable` function checks if the reason for a feature being unavailable is `missing_plan`. If the `jetpack_block_editor_enable_upgrade_nudge` filter is `false` then [the reason becomes `missing_plan__nudge_disabled`](https://github.com/Automattic/jetpack/blob/590d930bd476bef1f01aa2904a4b3d9adea58a5b/projects/plugins/jetpack/class.jetpack-gutenberg.php#L190) which will make `isUpgradable` return `false`

Reply:
I'm going to dig and confirm all scenarios here. Thanks, Paul 🙇

Reply:
I think you're right. I've updated the code removing that part.